cmd/coordinator.go

```diff
@@ -264,6 +272,251 @@ var estimateDepositsSweepFeeCommandDescription = "Estimates the satoshi " +
 	"legacy P2SH deposits. If the estimated fee exceeds the maximum fee " +
 	"allowed by the Bridge contract, the maximum fee is returned as result"
 
+var submitDepositSweepProofCommand = cobra.Command{
+	Use:              "submit-deposit-sweep-proof",
+	Short:            "submit deposit sweep proof",
+	Long:             submitDepositSweepProofCommandDescription,
+	TraverseChildren: true,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		ctx := cmd.Context()
+
+		_, tbtcChain, _, _, _, err := ethereum.Connect(
+			ctx,
+			clientConfig.Ethereum,
+		)
+		if err != nil {
+			return fmt.Errorf(
+				"could not connect to Ethereum chain: [%v]",
+				err,
+			)
+		}
+
+		btcChain, err := electrum.Connect(ctx, clientConfig.Bitcoin.Electrum)
+		if err != nil {
+			return fmt.Errorf("could not connect to Electrum chain: %v", err)
+		}
+
+		transactionHashFlag, err := cmd.Flags().GetString(transactionHashFlagName)
+		if err != nil {
+			return fmt.Errorf("failed to find transaction hash flag: %v", err)
+		}
```

Comment:
I can run the coordinator without the `--transaction-hash` provided:
```
./keep-client coordinator submit-deposit-sweep-proof --config configs/config.coordinator.toml
```

and it will not complain about the missing flag but fail later:
```
2023-06-19T11:17:48.268+0200    FATAL   keep-main       keep-core-2/main.go:29  failed to parse transaction hash flag: wrong hash string size
```

Is there a way to mark the flag as required and fail early? (cc @nkuba)

Reply:
Done in 9a874449fbbd767c62646bf2d3eb1055b3ee1f9c.
The flag is now marked as required.
Now the execution fails early:
```
2023-06-19T12:20:17.939+0200    FATAL   keep-main       keep-core/main.go:29    required flag(s) "transaction-hash" not set
```